lib/puppet/interface/documentation.rb

```diff
@@ -200,11 +201,11 @@ def short_description(value = nil)
     def author(value = nil)
       unless value.nil? then
         unless value.is_a? String
-          raise ArgumentError, 'author must be a string; use multiple statements for multiple authors'
+          raise ArgumentError, _('author must be a string; use multiple statements for multiple authors')
```

Comment:
Same as above for `author`, though again this is definitely debatable. I'd be fine merging this in as is

Reply:
yeah, changed